ui/src/services/salt/api.js

```diff
@@ -14,7 +14,7 @@ export function authenticate(user) {
   return saltApiClient.post('/login', {
     eauth: 'kubernetes_rbac',
     username: user.username,
-    password: user.token,
```

Comment:
So where did `user.token` go? Should it still be set? Does it have a different meaning now?

Reply:
In 2.4, UI relies on basic auth. I didn't check the `user` state management in details, but both `password` and `token` are set. The API changed however, `password` isn't expecting a token anymore, but the password itself.

Reply:
Yes, but the `token` attribute on this `user` object now still exists. For no good reason?

Reply:
Indeed, I don't think it's used. OK will remove it, and get some more conflicts, yay :trollface:!

Reply:
Got `git rerere` enabled, right?

Reply:
Done: 94251f5

Reply:
> Got `git rerere` enabled, right?

Yes of course, but now I'm changing things that I didn't before :)